### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/compliance/pci-dss/index.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/compliance/pci-dss/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/compliance/pci-dss/index?branch=pr-en-us-237) |
[docs/reference-architectures/virtual-machines-linux/multi-vm.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/reference-architectures/virtual-machines-linux/multi-vm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-linux/multi-vm?branch=pr-en-us-237) |
[docs/reference-architectures/virtual-machines-linux/single-vm.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/reference-architectures/virtual-machines-linux/single-vm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-linux/single-vm?branch=pr-en-us-237) |
[docs/reference-architectures/virtual-machines-windows/multi-vm.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/reference-architectures/virtual-machines-windows/multi-vm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/multi-vm?branch=pr-en-us-237) |
[docs/reference-architectures/virtual-machines-windows/single-vm.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/reference-architectures/virtual-machines-windows/single-vm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/single-vm?branch=pr-en-us-237) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C21%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201709211849537274-237%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NVRZQPGrtnuVy2FHbC%2F%2Fy0k8JqszgJLuBmNtblaTeKE%3D&st=2017-09-21T18%3A45%3A46Z&se=2017-09-22T18%3A55%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference-architectures/virtual-machines-linux/multi-vm.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ The architecture has the following components:
   * **Front-end configuration**. Associates the public IP address with the load balancer.
   * **Back-end address pool**. Contains the network interfaces (NICs) for the VMs that will receive the incoming traffic.
 * **Load balancer rules**. Used to distribute network traffic among all the VMs in the back-end address pool.
-* **VM Scale set**. A [VM scale set][vm-scaleset] is a set of identical VMs used to host a workload. Scale sets allow the number of VMs to be scaled in or out anually, or based on predefined rules.
-* **Availability set**. The [availability set][availability set] contains the VMs, making the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. In order for the SLA to apply, the availability set must include a minimum of two VMs. Availability sets are implicit in scale sets. If you create VMs outside a scale set, you need to create the availability set independently.
+* **VM scale set**. A [VM scale set][vm-scaleset] is a set of identical VMs used to host a workload. Scale sets allow the number of VMs to be scaled in or out annually, or based on predefined rules.
```
Comment (MikeWasson) [inline docs/reference-architectures/virtual-machines-linux/multi-vm.md]: annually -> manually

Unless it's a Black Friday thing 😸

File: docs/reference-architectures/virtual-machines-windows/multi-vm.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ The architecture has the following components:
   * **Front-end configuration**. Associates the public IP address with the load balancer.
   * **Back-end address pool**. Contains the network interfaces (NICs) for the VMs that will receive the incoming traffic.
 * **Load balancer rules**. Used to distribute network traffic among all the VMs in the back-end address pool.
-* **VM Scale set**. A [VM scale set][vm-scaleset] is a set of identical VMs used to host a workload. Scale sets allow the number of VMs to be scaled in or out anually, or based on predefined rules.
-* **Availability set**. The [availability set][availability set] contains the VMs, making the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. In order for the SLA to apply, the availability set must include a minimum of two VMs. Availability sets are implicit in scale sets. If you create VMs outside a scale set, you need to create the availability set independently.
+* **VM Scale set**. A [VM scale set][vm-scaleset] is a set of identical VMs used to host a workload. Scale sets allow the number of VMs to be scaled in or out annually, or based on predefined rules.
```
Comment (MikeWasson) [inline docs/reference-architectures/virtual-machines-windows/multi-vm.md]: annually -> manually

File: docs/reference-architectures/virtual-machines-linux/multi-vm.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/reference-architectures/virtual-machines-linux/multi-vm.md]: Oops - misread that typo :-)

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/compliance/pci-dss/index.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/compliance/pci-dss/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/compliance/pci-dss/index?branch=pr-en-us-237) |
[docs/reference-architectures/virtual-machines-linux/multi-vm.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/reference-architectures/virtual-machines-linux/multi-vm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-linux/multi-vm?branch=pr-en-us-237) |
[docs/reference-architectures/virtual-machines-linux/single-vm.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/reference-architectures/virtual-machines-linux/single-vm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-linux/single-vm?branch=pr-en-us-237) |
[docs/reference-architectures/virtual-machines-windows/multi-vm.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/reference-architectures/virtual-machines-windows/multi-vm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/multi-vm?branch=pr-en-us-237) |
[docs/reference-architectures/virtual-machines-windows/single-vm.md](https://github.com/mspnp/architecture-center/blob/abuck/typos/docs/reference-architectures/virtual-machines-windows/single-vm.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/virtual-machines-windows/single-vm?branch=pr-en-us-237) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C21%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201709211925105694-237%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mHchio5iIjNjJMj8ewxy77ggEz2erhjA5n0Ubj5md5M%3D&st=2017-09-21T19%3A21%3A03Z&se=2017-09-22T19%3A31%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**